packages/playground/src/pages/mocks/profile-settings/mockKeyList.ts

```diff
@@ -0,0 +1,22 @@
+import { KeysList } from '../../../components/profile-settings-keys-list'
+
+export const data: KeysList[] = [
+  {
+    created: 1728145770582,
+    verified: null,
+    identifier: 'try-1',
+    usage: 'auth',
+    fingerprint: 'SHA256:8H8leG+RViGZYLwzXx4N8htb122oFK2RK8RiMPqCgSk',
```

Comment:
use non PII info?

Reply:
Thats a public key...shouldn't be an issue